types/chart.js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Chart.js 2.7
+// Type definitions for Chart.js 2.8
```

Comment:
why is this change needed?

Reply:
From what I understood from the PR guide, minor version bump was needed.

Reply:
The chart.js module is currently at version 2.7.2, 2.8 doesn't exist yet. The guide makes suggestions about how to deal with breaking major version changes ie if chart.js was upgraded to v3.